src/common/legacy_config_opts.h

```diff
@@ -1459,12 +1459,41 @@ OPTION(rgw_override_bucket_index_max_shards, OPT_U32, 0)
 OPTION(rgw_bucket_index_max_aio, OPT_U32, 8)
 
 /**
- * whether or not the quota/gc threads should be started
+ * whether or not the quota threads should be started
  */
 OPTION(rgw_enable_quota_threads, OPT_BOOL, true)
+
+/**
+ * whether or not the gc threads should be started
+ */
 OPTION(rgw_enable_gc_threads, OPT_BOOL, true)
+
+/**
+ * whether or not the lc threads should be started
+ */
 OPTION(rgw_enable_lc_threads, OPT_BOOL, true)
 
+/**
+ * whether or not the bl threads should be started
+ */
+OPTION(rgw_enable_bl_threads, OPT_BOOL, true)
+
+OPTION(rgw_bl_max_objs, OPT_INT, 32)
+
+OPTION(rgw_bl_lock_max_time, OPT_INT, 60)  // total run time for a single bl processor work
+OPTION(rgw_bl_work_time, OPT_STR, "00:00-06:00") //job process bl at 00:00-06:00s
+/**
+ * bl thread need to upload opslog by sending http put requests to itself
+
+ * We need to support FCGI frontend which we can't find listening port
```

Comment:
@mikulely It would be good if we can use `can not` instead of `can't`

Reply:
will address when adapt bl to new config interface.